wp-content/themes/engage-2-x/templates/page-annual-report.twig

```diff
@@ -26,15 +26,15 @@
 			{% if main.acf_fc_layout == 'director' %}
 				<section class="ann__sct ann__sct_split wrapper--pad-huge">
 					<div class="ann__block flex_row">
-						<div class="ann__card"{% if main.background_color %} style="background-color:{{ main.background_color }};"{% endif %}>
```

Comment:
@DavidLungu Don't forget: we only want to _**add**_ code to this file. 

We don't want to alter the existing code. Even if it is just formatting, we want to just leave it as it is.